scene/main/timer.cpp

```diff
@@ -116,6 +116,10 @@ void Timer::stop() {
 	autostart = false;
 }
 
+bool Timer::is_time_left() const {
+	return get_time_left() > 0;
```

Comment:
I'm not sure if this would work properly, since it is floating-point comparison without epsilons...
Maybe `get_time_left() > 0.0001` would be more stable?